examples/pytorch/AxHyperOptimizationPTL/README.md

```diff
@@ -0,0 +1,64 @@
+## Ax Hyperparameter Optimization Example 
+In this example, we train a Pytorch Lightning model to classify [CIFAR-10 dataset](https://github.com/pytorch/vision/blob/master/torchvision/datasets/cifar.py). The code, adapted from this [repository](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html), is almost entirely dedicated to model training and hyperparameter optimization.
```

Comment:
Could you update `README.md` because we no longer train the model with the CIFAR-10 dataset?

Reply:
I will update the entire README.md file. Thanks

Reply:
Done. i have updated the README.md file